packages/blade/src/components/Tabs/TabList.web.tsx

```diff
@@ -97,20 +96,8 @@ const TabList = ({
                       })}
                 >
                   {variant === 'filled' && !isVertical
-                    ? React.Children.map(children, (child, index) => {
-                        return (
-                          <>
-                            {index > 0 ? (
-                              <Divider
-                                margin="auto"
-                                height="20px"
-                                variant="subtle"
-                                orientation="vertical"
-                              />
-                            ) : null}
-                            {child}
-                          </>
-                        );
+                    ? React.Children.map(children, (child) => {
+                        return <>{child}</>;
```

Comment:
Is this not same as `children`?

Reply:
We also won't need that if-else check itself if this is just children

Reply:
removed